app/DoctrineMigrations/Version20190826204730.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Wallabag\CoreBundle\Doctrine\WallabagMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190826204730 extends WallabagMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->skipIf($schema->hasTable($this->getTable('ignore_origin_user_rule')), 'It seems that you already played this migration.');
+        $this->skipIf($schema->hasTable($this->getTable('ignore_origin_instance_rule')), 'It seems that you already played this migration.');
```

Comment:
`but the table 'ignore_origin_user_rule' is missing. Very odd`

Reply:
The `skipIf` can be found in almost all migrations.
It can be seen as a way to prevent any accidental migration if doctrine loses the track of applied migrations.

Reply:
I implied that if we have the first table but not the second, there's something really fishy going on.

Reply:
@tcitworld should I stick each skipIf to its table creation or make only one skip with the two checks in the same statement?

Reply:
Maybe put stuff inside a `if` for both case and it'll be ok

```php
if (false === $schema->hasTable($this->getTable('ignore_origin_user_rule'))) {

}
// ...
if (false === $schema->hasTable($this->getTable('ignore_origin_instance_rule'))) {

}
```